### PR TITLE
[RFC] Update pvscheck.sh script

### PR DESCRIPTION
**Pull request by ZyX-I**
Blocks neovim/bot-ci#97

### CODE REVIEW

**Comment by ZyX-I** [comment]
Stopped travis for being useless

File: scripts/pvscheck.sh [hunk 1]
```diff
@@ -10,10 +10,17 @@ get_jobs_num() {
 
 help() {
   echo 'Usage:'
-  echo '  pvscheck.sh [target-directory [branch]]'
-  echo '  pvscheck.sh [--recheck] [target-directory]'
+  echo '  pvscheck.sh [--pvs URL] [target-directory [branch]]'
+  echo '  pvscheck.sh [--pvs URL] [--recheck] [target-directory]'
   echo '  pvscheck.sh --patch [--only-build]'
   echo
+  echo '    --pvs: Use the specify URL as a path to pvs-studio archive.'
```
Comment (brcolow) [inline scripts/pvscheck.sh]: specify -> specified

**Comment by ZyX-I** [comment]
Actually it appears that neovim/bot-ci#97 needs this: PVS guys no longer provide an old version.

**Comment by ZyX-I** [comment]
Quickbuild also recognizes `[ci skip]`?

**Comment by ZyX-I** [comment]
By the way, they have fixed the problem with vimvars.

**Pull request merged**